src/hooks/useLottery.ts

```diff
@@ -0,0 +1,82 @@
+import axios, { AxiosResponse } from "axios";
+import { useEffect, useState } from "react";
+import { Option } from "../components/atoms/Dropdown/Dropdown";
```

Comment:
I think it is a bit weird relying in an `Atom` here, let's remove this dependency.